examples/message/src/App.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
```

Comment:
General comment - this is rather complicated example. Some simpler (as short as possible) example would be fine as well. Also, this example is a good material for writing a test.